src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -9,12 +9,8 @@
 #include "overlays/gamestates/ovl_title/z_title.h"
 
 void MapSelect_LoadTitle(MapSelectState* this) {
-    {
-        GameState* gameState = &this->state;
-        gameState->running = false;
-    }
-
-    SET_NEXT_GAMESTATE(&this->state, Title_Init, TitleContext);
+    STOP_GAMESTATE(&this->state);
+    SET_NEXT_GAMESTATE(&this->state, ConsoleLogo_Init, sizeof(ConsoleLogoState));
 }
```

Comment:
I think `MapSelect_LoadTitle` should be updated too right? So `MapSelect_LoadConsoleLogo`?